src/firebase.js

```diff
@@ -0,0 +1,18 @@
+import { initializeApp } from 'firebase/app';
```

Comment:
cool w/ everything.. question about whether we want to put this in a db folder to keep things tidy